python_modules/dagster/dagster/core/definitions/graph_definition.py

```diff
@@ -950,7 +950,7 @@ def config_fn(x):
         field_aliases=inner_schema.field_aliases,
     )
 
-    config_evr = process_config(config_schema, default_config)
```

Comment:
whoops

Reply:
whoops what?

Reply:
just that we were processing and not only validating